mico-core/src/main/java/io/github/ust/mico/core/model/MicoService.java

```diff
@@ -19,26 +19,25 @@
 
```

Comment:
Again rearranged code or anything else?

Reply:
The imports were rearranged. The rest are real changes.